raspberry-pi/4/audio.nix

```
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.hardware.raspberry-pi."4".audio;
+in
+{
+  options.hardware = {
+    raspberry-pi."4".audio = {
+      enable = lib.mkEnableOption ''
+        Enable configuration for audio
```

```suggestion
        configuration for audio
```

*Whether to enable **Enable** configuration for audio* → *Whether to enable configuration for audio*.

`mkEnableOption` adds its own prefix to the description.

 - https://github.com/NixOS/nixpkgs/blob/069cd2bdc5303b9aafece12544138bac35160f7c/lib/options.nix#L98